src/main/java/com/salesforce/comdagen/model/Content.kt

```diff
@@ -0,0 +1,121 @@
+/*
+ *  Copyright (c) 2018, salesforce.com, inc.
+ *  All rights reserved.
+ *  SPDX-License-Identifier: BSD-3-Clause
+ *  For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+package com.salesforce.comdagen.model
+
+import com.salesforce.comdagen.RandomData
+import com.salesforce.comdagen.SupportedZone
+import com.salesforce.comdagen.config.ContentConfiguration
+
+
+/**
+ * Abstract content class which requires a seed and a contentConfig.
+ */
+abstract class AbstractContent(
```

Comment:
Can you imagine a use case for another child class than `RandomContent`? If not, merge these two.